exercises/two-bucket/canonical-data.json

```
@@ -142,4 +180,4 @@
       }
     }
   ]
-}
+}
```

```suggestion
}

```

The linter is complaining about the missing newline here.